en/appendices/4-5-migration-guide.rst

```diff
@@ -9,7 +9,7 @@ Upgrading to 4.5.0
 
 You can can use composer to upgrade to CakePHP 4.5.0::
 
-    php composer.phar require --update-with-dependencies "cakephp/cakephp:^4.5"
+    php composer.phar require --update-with-dependencies "cakephp/cakephp:4.5.x"
```

Comment:
Didnt we usually simply use `~4.5.0` ?

Reply:
I've switched to `.x` as it seems easier for devs who don't touch composer a lot. But, whatever the cake convention is, is fine with me.

Reply:
True, composer semver is a bit tricky to understand.